app/controllers/idv/phone_controller.rb

```diff
@@ -53,16 +38,24 @@ def phone_confirmation_required?
       idv_session.user_phone_confirmation != true
     end
 
+    def submit_proofing_attempt
+      idv_result = step.submit(step_params.to_h)
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION_VENDOR, idv_result.to_h)
+      redirect_to_next_step and return if idv_result.success?
+      handle_proofing_failure
+    end
```

Comment:
Noting for a follow-up PR that we will need to handle request timeouts here, similarly to what we did for Twilio. See JIRA issue LG-650.